### DISCUSSION TITLE
[Feature Request] Add CLI flag to allow debuggers to be attached to pyarmor compiled modules

### DISCUSSION THREAD

**Discussion by danbopes** [dashingsoft/pyarmor#1192] [Ideas]
I'm currently using a pyarmor module that has been compiled into a DLL via cffi. When running the code without a debugger attached, the code executes normally. If Visual Studio is attached to the process, the process simply exits with code 1, no errors or exceptions. See also issue #651 

I need to be able to debug the return values from the script, I couldn't care less about the contents of the module. Can we add command line flags to disable this functionality for users who wish their users to debug code? I cannot currently debug the functionality that this module offers currently, due to not being able to inspect the return values from the module.

**Comment by jondy** [comment]
This feature request is rejected, the obfuscated script is designed to quit if there is debugger.

If you're using Pyarmor 7.x, check documentation 7.x to obfuscate the script by feature `0`, it may work with debugger.
